.github/workflows/push.yml

```diff
@@ -135,6 +136,21 @@ jobs:
           VERSION: ${{ steps.generate_operator_tag.outputs.new_tag }}
         run: make update-resources-ci
 
+      # Publish helm chart
```

Comment:
Please add step for helm lint here as well.

Reply:
done